examples/zkapps/anonymous-message-board/src/Add.test.ts

```diff
@@ -0,0 +1,67 @@
+import { Add } from './Add';
```

Comment:
Can we delete this? It shouldn't have anything to do with the message board tutorial